webapp/src/components/settings/WakuForm.tsx

```diff
@@ -16,8 +22,15 @@ const mapWakuState = (status: WakuState | undefined): string => {
 	}
 };
 
-function WakuForm() {
+interface WakuFormProps {
+	currentSettings: Partial<SettingsFormData>;
+}
+
+function WakuForm({
+	currentSettings: { harbourAddress, rpcUrl },
+}: WakuFormProps) {
 	const [status, setStatus] = useState<WakuState>();
+	const [supported, setSupported] = useState<boolean | null>(null);
```

Comment:
why use `null` here? With the current usage having the default as `false` would do the same. `null` would only make sense if we have a "loading" state, right?